### ISSUE TITLE
[v3.1.0] Register Error

### ISSUE THREAD

**Issue by FGServers** [XGProyect/XG-Proyect-v3.x.x#229]
**Describe the bug**
When you put the data and give it a recorder, it throws three errors this time I can see them thanks to the fact that I create a file logs. I'm doing the test on localhost and with xampp

The version that installs the 478 commit new installation

**To Reproduce**
1. Go to 'register'

**Expected behavior**
Well, I think it's a complicated error of the registry of the registry of the errors but the registry of the error of the registry of the user.

Log 1: php-error-20180623-1529804466-157d36bce76fc8c62a435dd95a5368a22c07b780
`|::1|php|2|arsort() expects parameter 1 to be array, null given|Where called: line 205 of  <br/>(in C:\xampp\htdocs\application\libraries\Statistics_library.php)|2018/06/23 22:41:06|`
Log 2: php-error-20180623-1529804467-053e2bb0ccd4a514cbc89d11e60278fd6eded00f
`|::1|php|2|arsort() expects parameter 1 to be array, null given|Where called: line 205 of  <br/>(in C:\xampp\htdocs\application\libraries\Statistics_library.php)|2018/06/23 22:41:07|`
Log 3: php-error-20180623-1529804496-f2a2abd41133d63d99930ded7e87a301e3d24c63
`|::1|php|2|mail(): SMTP server response: 550 Access denied - Invalid HELO name (See RFC2821 4.1.1.1)|Where called: line 1845 of  <br/>(in C:\xampp\htdocs\system\libraries\Email.php)|2018/06/23 22:41:36|`

**Screenshots**
![image](https://user-images.githubusercontent.com/2124810/41815118-42ac7a12-7737-11e8-9922-19ebb2c3a6df.png)

**Desktop (please complete the following information):**
 - OS: Windows 10 - 1803 Compilacion 17134.112
 - Browser Chrome
 - Version Versión 67.0.3396.87 (Build oficial) (64 bits)

**Smartphone (please complete the following information):**
 - Device: Samsung Galaxy s9
 - OS: Android 8.1.1
 - Browser Chrome
 - Version 67.0.3396.87

**Additional context**
Don't take it's bad as sometimes every time i report something but the are suggestions too old users we want to help you and the proyect.

1. This is more personal it is necessary all this to report bug ..
As I say it is a more personal question to understand. 
2. All the errors of the update of the last commits

**Comment by FGServers** [comment]
Lucky este error solo pasa en localhost lo acabo de comprobar en el hosting y mira me llega el correo


Welcome to XG Proyect, thanks for joining our game!In this email you'll find all the information that you need to login into your account.
--
 
Data to access the game
Your username: jonamix
Your password: **********
Game link: ***********
 
Enjoy the game!
 
Regards,The XG Proyect Team.-

yo coloque los datos en el email.php esta bien eso

**Comment by LucasKovacs** [comment]
Jona, si es alguna configuración en el local. No lo cerre porque queria investigarlo y agregar algun exception

**Comment by sandorfeher00** [comment]
Hi!
please help me!
I trying using your project 3.0.4
but im getting this when I try to register in-game
XG Proyect
500. That’s an error.

The requested URL throw an error. Contact the game Administrator. That’s all we know.
thank you

**Comment by BeReal86** [comment]
Hi, @sandorfeher00 thanks for your feedback.

which php version do you use?

**Comment by sandorfeher00** [comment]
Hi! Thanks for your replay.
At the moment I'm using 7.2
I have tryed the newest version of xg project
3.1 version which is work
I can register.
But I fund many bugs like metal Krystal miner
Is not going up from lvl 13
But take the resource.
And when I send the coloniser ship
To the plenet is get 500 error.
So that's why I trying the stable version but I cant register.
If I going in the game from Admin panel
I can play but I just made the coloniser ship and everything is scalloped
500 error.
